Vizzuality/heco-invest
# feat: Validate images with help of mini_magick

Making sure that uploaded pictures are truly images. API returns url of images only if
 - image was analyzed with AnalyzeJob
 - image contains dimensions --> therefore it is image
 
Till we are sure that image is truly image, we return `null` value.

## Testing instructions
Testing is a bit tricky till there is no way how to run background jobs, but basically it can be done with:
- create or update picture of project developer via swagger doc
- check that null is returned because image is not analyzed yet
- open console and manually run  `ActiveStorage:: AnalyzeJob.perform_now(YOUR_BLOB)`
- check that project developer info returned by API contains picture urls now

## Tracking

https://vizzuality.atlassian.net/browse/LET-226
